.github/workflows/sign.yml

```diff
@@ -34,162 +26,38 @@ jobs:
   Sign:
     runs-on: windows-latest
     steps:
+    - name: Checkout repository
```

Comment:
Do we still need to checkout the repo now that we're downloading the script file?

Reply:
Nope, I'll remove it.